.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Feature requests and suggestions
+    url: https://github.com/nextcloud/news/discussions
```

Comment:
I've made the exact same suggestion in the maintainer chat 10 minutes before your PR. 😃 

It would be better to link directly to the Features category: https://github.com/nextcloud/news/discussions/categories/features

Also please sign your commit by using the following commands:
```sh
git commit -s --amend
git push --force-with-lease
```

Reply:
> Also please sign your commit by using the following commands

I'm doing this from the GH website - I don't think that's possible, but my [commits to AntennaPod](https://github.com/AntennaPod/AntennaPod/pull/4740) seem to be signed still (?)

> It would be better to link directly to the Features category: https://github.com/nextcloud/news/discussions/categories/features

Will do that, and add a direct link to feed issue then also.